docs/development/maintenance.md

```diff
@@ -13,104 +15,124 @@ modules of ILIAS.
-	* "[Coordinator](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/maintenance-coordinator.md#coordinator-model)".
-* **"first_maintainer"**: One entry in the form `<username> (<userid>)` pointing to a valid user on 
-	https://docu.ilias.de. Only relevant if **the maintenance_model** is set to "Classic".
-* **"second_maintainer"**: One entry in the form `<username> (<userid>)` pointing to a valid user on https://docu.ilias.de. 
-	Only relevant if **the maintenance_model** is set to "Classic".
-* **"[implicit_maintainers](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/maintenance.md#implicit-maintainers)"**: 
-    An array in the form [ `<username> (<userid>)` ] pointing to valid users on https://docu.ilias.de. Only relevant if 
-    **the maintenance_model** is set to "Classic" **and** neither a first nor a second maintainers is set.
-* **"coordinator"**: An array in the form [ `<username> (<userid>)` ] pointing to valid users on https://docu.ilias.de.
-	Only relevant if **the maintenance_model** is set to "Coordinator".
-* **"[pr_management](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/maintenance.md#additional-competences)"** : 
-    An array in the form [ `<username> (<userid>)` ] pointing to valid users on https://docu.ilias.de.
-* **"[issue_management](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/maintenance.md#additional-competences)"** :
-  An array in the form [ `<username> (<userid>)` ] pointing to valid users on https://docu.ilias.de.
-* **"tester"**: One entry in the form `<username> (<userid>)` pointing to a valid user on
-  https://docu.ilias.de.
-* **"testcase_writer"**: One entry in the form `<username> (<userid>)` pointing to a valid user on
-  https://docu.ilias.de.
+# Authorities
+The ILIAS-Community strives to create and maintain a secure, reliable, and
+adaptable learning management. We foster participation by a diverse set of
+developers, designers, testers and other contributors, but we also have to guarantee the sustainability and the quality
+of the ILIAS source code.
+
+To make sure people with diverse backgrounds and capabilities can participate
+in our Community and contribute to the development of ILIAS and its code base,
+we split the code into units (often called components, even though the term
+is hard to define) and we define a set of authorities community members can have
+concerning these units of code. We understand an authority as the counterpart of
+a responsibility: The people having the authorities to do something in a unit of
+code also assume the responsibility for the corresponding functions.
+
+For the context of ILIAS, we define **four** different authorities:
+
+1. **Authority to Sign of Code Changes**
+2. **Authority to Sign of Conceptual Changes**
+3. **Authority to Sign of Testcase Changes**
+4. **Authority to (De-)Assign Authorities**
```

Comment:
Man _könnte_ bei 4. umbenennen in etwas wie „Authority to Modify Authority Assignments“, falls irgendwo die Klammer stört (entweder den Lesefluss oder eine Eingabe, wo besser keine speziellen Zeichen stehen sollten). Ist aber nicht sonderlich wahrscheinlich, glaube ich, also wohl eher optische Sache. Für mich ist der Begriff auch mit Klammerung in Ordnung. Bei gesprochener Kommunikation ist es evtl. etwas komisch, insbesondere falls jemand diese 4 Möglichkeiten nicht kennt.

Reply:
Ich behalte einmal die ursprüngliche Formulierung. Habe hier aber keine Präferenz. Mündlich würde ich das (de-) einfach weglassen.

Reply:
Ah, eine schöne und einfache Lösung. Finde ich gut! :+1: